### PR TITLE
Update power control docs.

### PR DESCRIPTION
**Pull request by johnsonw**
Update power control docs.

Signed-off-by: Will Johnson <william.c1.johnson@gmail.com>

### CODE REVIEW

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+Power control can be configured once all of the managed servers have been added successfully and the network interfaces have been updated. To do this, ensure `/usr/local/bin` is added to PATH in ~/.bashrc:
 
+```bash
+ export PATH="$PATH:/usr/local/bin"
 ```
+
+Next, install `fence-agents-vbox` on the `adm` node and on all of the server nodes:
+
+```bash
 vagrant sh -c "sudo yum install -y fence-agents-vbox" adm oss1 oss2 mds1 mds2
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: I don't think sudo is necessary.

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
 # <a name="Top"></a>Setting up Power Control
 
-Power control can be configured once all of the managed servers have been added successfully and the network interfaces have been updated. To do this, install `fence-agents-vbox` on the `adm` node and on all of the server nodes:
+Power control can be configured once all of the managed servers have been added successfully and the network interfaces have been updated. To do this, ensure `/usr/local/bin` is added to PATH in ~/.bashrc:
```
Comment (jgrund) [inline docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: Make sure to note this needs to be done on the host node. Also, probably worth noting it needs to be somewhere that is picked up by the ssh session, as .bashrc is a noop on OS X iirc.

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md (hunk 1 above)
Comment (johnsonw) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: We may need to ssh in as root then?
```
 2018-06-25 11:47:07 ⌚ |ruby-2.4.1| DDN11359m in ~/vagrant_projects/vhpc
○ → vagrant sh -c "yum install -y fence-agents-vbox" adm
adm::
Loaded plugins: fastestmirror, langpacks, versionlock
Repository extras is listed more than once in the configuration
You need to be root to perform this command.
```

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md (hunk 2 above)
Comment (johnsonw) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: I updated this block of text to be more descriptive. I think .bash_profile is loaded when logging into a session and .bashrc is loaded for sessions that you don't have to log into. I'm not entirely sure if that's correct. In my case, .bash_profile is loaded and at the bottom I load my .bashrc, which is where I export most of my environment variables:

```
if [ -f ~/.bashrc ]; then
   . ~/.bashrc
fi
```

Comment (jgrund) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: It's worth testing if that's actually the case if you remove that block from your `.bash_profile`

File: docs/Contributor_Docs/cd_Setting_Up_Power_Control.md (hunk 1 above)
Comment (jgrund) [reply docs/Contributor_Docs/cd_Setting_Up_Power_Control.md]: yes, you're right. For some reason I thought this happened as root

**Pull request merged**